scijava/scijava-ops-engine/src/test/java/org/scijava/ops/engine/impl/ProvenanceTest.java

```diff
@@ -246,14 +246,20 @@ public void testDependencylessOpRecoveryFromString() {
 	 */
 	@Test
 	public void testOpWithDependencyRecoveryFromString() {
-		OpInfo info = singularInfoOfName("test.provenanceMapper");
-		OpInfo dependency = singularInfoOfName("test.provenanceMapped");
-		InfoChain depChain = new InfoChain(dependency);
-		InfoChain chain = new InfoChain(info, Collections.singletonList(depChain));
-		String signature = chain.signature();
-		Nil<Function<Double[], Thing>> special = new Nil<>() {};
-		@SuppressWarnings("unused")
-		Function<Double[], Thing> actual = ops.opFromSignature(signature, special);
+		// Get the Op
+		Function<Double[], Thing> mapper = ops //
+				.op("test.provenanceMapper") //
+				.input(new Double[] {5.0, 10.0, 15.0}) //
+				.outType(Thing.class) //
+				.function();
+		// Get the signature from the Op
+		String signature = history.signatureOf(mapper);
+		// Generate the Op from the signature and an Op type
+		Nil<Function<Double, Thing>> specialType = new Nil<>() {};
+		Function<Double, Thing> actual = ops //
+				.opFromSignature(signature, specialType);
+		// Assert Op similarity
+		Assertions.assertTrue(wrappedOpEquality(mapper, actual));
```

Comment:
I've been wondering... we used to use `import static` with JUnit 4. Now that we are on JUnit 5, would it make sense to do the same, statically importing used `Assertions` methods? Or do you prefer the verbosity this way? It's more consistent with everything else, but I ask because it's a break from previous JUnit convention.